tutorials/workflow/csharp/challenges-tips/VersioningWorkflow.cs

```diff
@@ -0,0 +1,48 @@
+using Dapr.Workflow;
+
+namespace WorkflowApp;
+
+/// <summary>
+/// This is the initial version of the workflow.
+/// Note that the input argument for both activities is the orderItem (string).
+/// </summary>
+internal sealed class VersioningWorkflow1 : Workflow<string, int>
+{
+    public override async Task<int> RunAsync(WorkflowContext context, string orderItem)
+    {
+        int resultA = await context.CallActivityAsync<int>(
```

Comment:
It's far more common for developers to use `var` as in `var resultA` instead of repeating the type.